examples/openlayers/vectors/geographic-epsg4326-vector-basic.html

```diff
@@ -0,0 +1,43 @@
+<!doctype html>
+<!--
+   GIBS Web Examples
+
+   Copyright 2013 - 2018 United States Government as represented by the
```

Comment:
I guess the copyright should be changed to 2020?

Reply:
Yeah, that makes sense to me. 👍  I think we removed these banners in favor of keeping them in one place for WV a while back. Something to consider here as well... But for now I updated them ©️